packages/blocks/cloudfix/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { createCustomApiCallAction } from '@openops/blocks-common';
+import { createBlock, Property } from '@openops/blocks-framework';
+import { BlockCategory } from '@openops/shared';
+import { cloudfixAuth } from './lib/common/auth';
+
+const markdown = `
+For more information, visit the [CloudFix API documentation](https://docs.cloudfix.com/reference/introduction).
```

Comment:
is this the right URL? seems broken

Reply:
nope sorry, that was a placeholder, as far as i understand there is no API documentation